src/debugAdapter/goDebug.ts

```diff
@@ -169,6 +169,14 @@ class Delve {
 				return;
 			}
 			let dlv = getBinPath('dlv');
+
+			// If dlv not found, try using the GOPATH that was set in env in launch.json
+			if (!existsSync(dlv) && env['GOPATH'] && !process.env['GOPATH']) {
+				process.env['GOPATH'] = env['GOPATH'];
+				dlv = getBinPath('dlv');
+				process.env['GOPATH'] = undefined;
```

Comment:
Wouldn't it be nicer if the getBinPath took an (optional) second argument called gopath and used that instead of changing the process.env['GOPATH']?